### PR TITLE
Release/ADAPT-000 Gallery/Campaign page logo fix

### PR DESCRIPTION
**Pull request by yvonnetangsu**
# READY FOR REVIEW
- (Edit the above to reflect status)

# Summary
- Prepare release with fixes that Patrick requested to go live before 11/20

# Review By (Date)
- When does this need to be reviewed by?

# Criticality
- How critical is this PR on a 1-10 scale? Also see [Severity Assessment](https://stanfordits.atlassian.net/browse/D8CORE-1705).
- E.g., it affects one site, or every site and product?

# Review Tasks

## Setup tasks and/or behavior to test

1. Check out this branch
2. Navigate to...
3. Verify...

## Front End Validation
- [ ] Is the markup using the appropriate semantic tags and passes HTML validation?
- [ ] Cross-browser testing has been performed?
- [ ] Automated accessibility scans performed?
- [ ] Manual accessibility tests performed?
- [ ] Design is approved by @ user?

### Code
- [ ] Are the naming conventions following our standards?
- [ ] Does the code have sufficient inline comments?
- [ ] Is there anything in this code that would be hidden or hard to discover through the UI?
- [ ] Are there any [code smells](https://blog.codinghorror.com/code-smells/)?
- [ ] Are tests provided? eg (unit, behat, or codeception)

### Code security
- [ ] Are all [forms properly sanitized](https://www.drupal.org/docs/8/security/drupal-8-sanitizing-output)?
- [ ] Any obvious [security flaws or new areas for attack](https://www.drupal.org/docs/8/security)?

## General
- [ ] Is there anything included in this PR that is not related to the problem it is trying to solve?
- [ ] Is the approach to the problem appropriate?

# Affected Projects or Products
- Does this PR impact any particular projects, products, or modules?

# Associated Issues and/or People
- JIRA ticket(s)
- Other PRs
- Any other contextual information that might be helpful (e.g., description of a bug that this PR fixes, new functionality that it adds, etc.)
- Anyone who should be notified? (`@mention` them here)

# Resources
- [AMP Tool](https://stanford.levelaccess.net/index.php)
- [Accessibility Manual Test Script](https://docs.google.com/document/d/1ZXJ9RIUNXsS674ow9j3qJ2g1OAkCjmqMXl0Gs8XHEPQ/edit?usp=sharing)
- [HTML Validator](https://validator.w3.org/)
- [Browserstack](https://live.browserstack.com/dashboard) and link to [Browserstack Credentials](https://asconfluence.stanford.edu/confluence/display/SWS/External+Account+Credentials)

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -2,7 +2,7 @@
   "name": "giving-to-stanford",
   "private": true,
   "description": "Giving to Stanford",
-  "version": "0.1.0",
+  "version": "0.1.1",
```
Comment (yvonnetangsu) [inline package.json:5]: Not super familiar the the release process on Giving. LMK if this looks right 🤔

Comment (sherakama) [reply package.json:5]: I was just checking the branch diff. S'all good.

Comment (yvonnetangsu) [reply package.json:5]: Awesome, thanks @sherakama !

**Comment by yvonnetangsu** [comment]
@sherakama Thanks for adding the workflow files! Is there a chance we could roll this out today? I'm thinking that since this is going live this Saturday and I'm out next week, it might be good roll this out a day earlier so we have tomorrow to fix anything if needed 🤔

**Comment by sherakama** [comment]
> @sherakama Thanks for adding the workflow files! Is there a chance we could roll this out today? I'm thinking that since this is going live this Saturday and I'm out next week, it might be good to roll this out a day earlier so we have tomorrow to fix anything if needed 🤔

Let's dual release today during our 1pm session.

**Comment by yvonnetangsu** [comment]
> > @sherakama Thanks for adding the workflow files! Is there a chance we could roll this out today? I'm thinking that since this is going live this Saturday and I'm out next week, it might be good to roll this out a day earlier so we have tomorrow to fix anything if needed 🤔
> 
> Let's dual release today during our 1pm session.

Awesome - thanks, @sherakama ! 😄

**Review by sherakama — APPROVED** [review]

**Pull request merged**